.github/workflows/prepare_raven_release.yml

```diff
@@ -1,10 +1,7 @@
-name: Prepare Release
```

Comment:
I'd prefer keeping it for now for releasing new integrations in the future.

Reply:
We can add this back easily. I'd prefer if we nudge people to use the batch release. What do you think?

Reply:
ok let's drop it for now 👍